device/arista/x86_64-arista_7280dr3a_36/platform_env.conf

```diff
@@ -1,4 +1,5 @@
 SYNCD_SHM_SIZE=512m
 dmasize=64M
 usemsi=1
+macsec_enabled=1
```

Comment:
MACsec is not supported for this specific SKU, it should only be enabled for the 7280dr3am and 7280dr3ak SKUs.